### DISCUSSION TITLE
Hope it can keep the location and taking time of the photo; add the batch selection feature.

### DISCUSSION THREAD

**Discussion by FlakyyyCroissant** [localsend/localsend#474] [Discussion (English, International)]
Now after transfering, the information of the photos would show the trasfering time, and the location info disapear. But sometimes the shooting time and location would help us organise them. The tools like Landrop can keep these info. Hope localsend could do that too!

Furthermore, when we transfer a group of photos, localsend can only click them one by one. Instead of quick choose or choose all in an album. It takes time and sometimes easy to miss.

If these two features can be solve, it will be the perfect tool that I ever met!

**Comment by k3b** [comment]
> after transfering, the information of the photos ... the location info disapear. 

Try sending the photo from a filemanger instead of sending from a gallery app.

Technical explanation:
starting with android-10 google has implemented "removing gps-location-data from photo" as a "security feature" when the photo is read via a mediaapi uri (i.e from inside a gallery app).
